tests/integration/targets/mso_schema_template_bd/tasks/main.yml

```diff
@@ -405,13 +405,31 @@
       name: VRF
       schema: '{{ mso_schema | default("ansible_test") }}'
       template: Template1
+    intersite_bum_traffic: true
+    arp_flooding: true
+    ep_move_detection_mode: true
   register: nm_add_bd_4
 
+- name: Add bd 4 (normal mode) with no ep_move_detection_mode
+  cisco.mso.mso_schema_template_bd:
+    <<: *bd_present
+    schema: '{{ mso_schema | default("ansible_test") }}'
+    template: Template1
+    bd: ansible_test_4
+    vrf:
+      name: VRF
+      schema: '{{ mso_schema | default("ansible_test") }}'
+      template: Template1
+    ep_move_detection_mode: false
+  register: nm_add_bd_4_no_ep
+
 - name: Verify nm_add_bd_2 and nm_add_bd_3
   ansible.builtin.assert:
     that:
     - nm_add_bd_2 is changed
     - nm_add_bd_3 is changed
+    - nm_add_bd_4 is changed
```

Comment:
for nm_add_bd_2 and nm_add_bd_3, should we assert the default?

Reply:
As discussed, will leave this as is